<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listado de Comunas de Chile</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblComunas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Comuna :</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="listaComunas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>1121</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>570</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="txtBuscar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>301</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>LISTA DE COMUNAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>600</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/save.png</normaloff>../icon/save.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/add-3.png</normaloff>../icon/add-3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRefrescar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>600</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Refrescar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/repeat.png</normaloff>../icon/repeat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>600</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/garbage-1.png</normaloff>../icon/garbage-1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>600</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/edit.png</normaloff>../icon/edit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>600</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/exit-2.png</normaloff>../icon/exit-2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>600</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/error.png</normaloff>../icon/error.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/binoculars.png</normaloff>../icon/binoculars.png</iconset>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExportarPDF">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>100</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/binoculars.png</normaloff>../icon/binoculars.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImprimir">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>100</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Imprimir</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/binoculars.png</normaloff>../icon/binoculars.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVistaPrevia">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>100</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Vista Previa</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/binoculars.png</normaloff>../icon/binoculars.png</iconset>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidgetComunas">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>451</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <zorder>treeWidgetComunas</zorder>
   <zorder>lblComunas</zorder>
   <zorder>txtBuscar</zorder>
   <zorder>label</zorder>
   <zorder>btnGuardar</zorder>
   <zorder>btnAgregar</zorder>
   <zorder>btnRefrescar</zorder>
   <zorder>btnEliminar</zorder>
   <zorder>btnModificar</zorder>
   <zorder>btnSalir</zorder>
   <zorder>btnCancelar</zorder>
   <zorder>btnBuscar</zorder>
   <zorder>btnExportarPDF</zorder>
   <zorder>btnImprimir</zorder>
   <zorder>btnVistaPrevia</zorder>
   <zorder>listaComunas</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
